<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #22222e
</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E90FF</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/BankLogo.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="hello_text">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Добро пожаловать!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profile_button">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/profile.png</normaloff>../images/profile.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="us_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="operations_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1E90FF;
	border-radius: 30;
	color: white
}

QPushButton:pressed {
	background-color:	#6495ED;
}</string>
    </property>
    <property name="text">
     <string>Операции с деньгами</string>
    </property>
   </widget>
   <widget class="QPushButton" name="history_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1E90FF;
	border-radius: 30;
	color: white
}

QPushButton:pressed {
	background-color:	#6495ED;
}</string>
    </property>
    <property name="text">
     <string>История переводов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="value_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1E90FF;
	border-radius: 30;
	color: white
}

QPushButton:pressed {
	background-color:	#6495ED;
}</string>
    </property>
    <property name="text">
     <string>Конвертер валюты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="credits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1E90FF;
	border-radius: 30;
	color: white
}

QPushButton:pressed {
	background-color:	#6495ED;
}</string>
    </property>
    <property name="text">
     <string>Калькулятор кредитов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1E90FF;
	border-radius: 30;
	color: white
}

QPushButton:pressed {
	background-color:	#6495ED;
}</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #6495ED
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="card_logo1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/debet_card.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="card_name1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Дебетовая карта</string>
     </property>
    </widget>
    <widget class="QLabel" name="card_balans1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Баланс:</string>
     </property>
    </widget>
    <widget class="QLabel" name="balans_info1">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="balans_info2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="free_money">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1E90FF;
	border-radius: 30;
	color: white
}

QPushButton:pressed {
	background-color:	#6495ED;
}</string>
    </property>
    <property name="text">
     <string>Набрать другу</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
